build/configs/artik053/STDK/defconfig

```diff
@@ -436,7 +436,7 @@ CONFIG_ADC=y
 CONFIG_ADC_FIFOSIZE=8
 CONFIG_ADC_NPOLLWAITERS=2
 # CONFIG_DAC is not set
-# CONFIG_DRIVERS_KERNEL_TEST is not set
+# CONFIG_OS_KERNEL_TEST is not set
```

Comment:
CONFIG_OS_KERNEL_TEST still refers to kernel test. 
Better to change it as CONFIG_OS_API_TEST

Reply:
done